packages/backend-common/src/reading/GitlabUrlReader.ts

```diff
@@ -311,6 +311,11 @@ export class GitlabUrlReader implements UrlReader {
       `${
         pathToProject.origin
       }${relativePath}/api/v4/projects/${encodeURIComponent(project)}`,
+      {
+        headers: {
+          ...getGitLabRequestOptions(this.integration.config).headers,
+        },
+      },
```

Comment:
```suggestion
      getGitLabRequestOptions(this.integration.config),
```

No need to extract and spread just the headers here right? should be able to use the options directly

Reply:
Agreed. Change made!